lib/sqlalchemy/dialects/postgresql/base.py

```diff
@@ -2307,6 +2316,29 @@ def visit_create_index(self, create, **kw):
 
         return text
 
+    def visit_unique_constraint(self, constraint, **kw):
```

Comment:
**mike bayer** (zzzeek) wrote:

I think I want to make this only def define_unique_constraint_distinct(self, constraint, **kw) and not rewrite all of visit_unique_constraint(), see below

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4632

Reply:
Done